src/opnsense/mvc/app/models/OPNsense/Unbound/Unbound.xml

```diff
@@ -1,7 +1,7 @@
 <model>
     <mount>//OPNsense/unboundplus</mount>
     <description>Unbound configuration</description>
-    <version>1.0.9</version>
+    <version>1.0.10</version>
```

Comment:
Don't need this unless we deal with a new default value, but the new value is not required.
```suggestion
    <version>1.0.9</version>
```